pkg/metrics/dapr_metrics.go

```diff
@@ -0,0 +1,95 @@
+package metrics
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"contrib.go.opencensus.io/exporter/prometheus"
+	ocprom "contrib.go.opencensus.io/exporter/prometheus"
+	"github.com/dapr/dapr/pkg/logger"
+	prom "github.com/prometheus/client_golang/prometheus"
+	"go.opencensus.io/stats/view"
+)
+
+const (
+	// DefaultMetricNamespace is the prefix of metric name
+	DefaultMetricNamespace = "dapr"
+	defaultMetricsPath     = "/"
+)
+
+// DaprMetricExporter is the metric exporter for dapr
+type DaprMetricExporter struct {
+	exporter  *prometheus.Exporter
```

Comment:
Should we take exporter behind interface to avoid tight coupling with prometheus..?

Reply:
Basically I would like to follow [this style](https://github.com/golang/go/wiki/CodeReviewComments#interfaces). We can add interface later when we support the other exporters.

Reply:
I did some refactoring to use interfaces :) including naming...